tests/e2e/global-setup.js

```diff
@@ -1,53 +1,16 @@
-/*
```

Comment:
Dont remove the header, update it :-)

Reply:
This is a new file, the old one still exists under `tests/electron`.